code/modules/clothing/clothing.dm

```diff
@@ -438,24 +438,24 @@ BLIND     // can't see anything
 	name = "suit"
 	var/fire_resist = T0C+100
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
-	allowed = list(/obj/item/weapon/tank/emergency_oxygen)
+	allowed = list(/obj/item/weapon/storage/pouch/,
+		/obj/item/weapon/gun,
+		/obj/item/weapon/melee,
+		/obj/item/weapon/material/kitchen,
+		/obj/item/weapon/material/knife,
+		/obj/item/weapon/material/butterfly,
+		/obj/item/ammo_magazine,
+		/obj/item/ammo_casing,
+		/obj/item/weapon/handcuffs,
+		/obj/item/weapon/tank/emergency_oxygen,
+		/obj/item/weapon/cell,
+		/obj/item/device/lighting/toggleable/flashlight)
```

Comment:
This is just reverting a lot of my recent work, no please.

Suit storage is a very large slot for something strapped over your torso, it should hold almost anything, weapon/material and weapon/tank were fine

revert changes to this list please

Reply:
Ah, whoops. Was wondering why there was two lists.